samples/extensions/README.md

```diff
@@ -20,3 +20,72 @@ $ bazel run //src/workerd/server:workerd -- serve $(pwd)/samples/extensions/conf
 $ curl localhost:8080 -X POST -d 'veggie'
 9
 ```
+
+
+## Demonstrated Methods
+
+### Importable Module
+
+#### Usage
+The module `burrito-shop:burrito-shop`, which is defined in the config [burrito-shop.capnp](burrito-shop.capnp), is an importable module.
+
+You can import it as shown below:
+
+```javascript
+import { BurritoShop } from "burrito-shop:burrito-shop";
+```
+
+#### Definition
+This import definition is demonstrated in [burrito-shop.js](burrito-shop.js).
+As shown in 
+
+### Environment Variable (with internal initialization from environment variables)
+
+#### Usage
+
+The module `burrito-shop:binding` is defined in the config [burrito-shop.capnp](burrito-shop.capnp) and is provided as an environment variable.
+
+You can access it as shown below in [worker.js](worker.js) provided the modules name is shop(as defined in the bindings section of [config.capnp](config.capnp)
+
+```javascript
+env.shop
+```
+
+The initialization is demonstrated in [binding.js](binding.js).
+
+##### Definition
```

Comment:
Inconsistent level, other sections have four '#' levels

Reply:
Fixed the inconsistent header levels.